manifests/server.pp

```diff
@@ -29,7 +29,7 @@
 	$ips = false,	# an optional list of ip's for each in hosts[]
 	$clients = []	# list of allowed client ip's	# TODO: get from exported resources
 ) {
-	$FW = '$FW'			# make using $FW in shorewall easier
+	$fw = '$FW'			# make using $FW in shorewall easier
```

Comment:
Is there a reason to use a lowercase varname? Is it required in puppet 4? The reason I left it as caps is that this way the shorewall rules are perfectly readable as folks are used to them.